### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by rolandfbk**
I have built the basic UI on the main.rb file.
When you run the main.rb file on the console, you will have choices to:

1. list all books.
2. list all people.
3. create a student or a teacher
4. create a book.
5. create a rental.
6. list all rentals for a given person's id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+  def message_menu
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
```
Comment (dasileker) [inline app.rb]: - [ ] Hello, excellent job on adding the UI, but as the requirement say that the `App.rb` will contain the logic and the  `main.rb` will have the start app with the interface function in the app.rb file

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @rolandfbk,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Linters are green ✅.
- The readme file is good 👌.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (rolandfbk) [reply app.rb]: May I please call you on zoom so that you can help me understand the concept?

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @rolandfbk 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**